README.md

```diff
@@ -234,6 +234,25 @@ the roles listed above.
             repository_username: "my_user"
             repository_password: "my_password"
 ```
+## Plugin usage
+
+### Inventory plugin
+
+The inventory plugin allows you to retrieve your inventory (list of Rudder nodes) in the Ansible inventory.
+
+#### Plugin default parameters
+- `cache`: Enable caching (default: `False`)
```

Comment:
I did not see where those were configurable in the code, are they? You often do not want to run the inventory from the Rudder server but will instead run it from your personal workstation.
The api uri also needs to be configurable.

Reply:
I just corrected it so that the values are properly configurable.

Reply:
:+1: Looks good, just add an example in the README with a proper inventory call with override of the uri for instance.

Reply:
It is now done !